internal/runner/runner.go

```diff
@@ -206,6 +206,7 @@ func (r *Runner) prepareInput() error {
 		var hosts = []string{item}
 		if r.options.WordList != "" {
 			var subdomain string
+			hosts = nil
```

Comment:
Looks good. I want to propose this little refactor as there are a lot of nested if:
```go
var hosts []string
switch {
    case r.options.WordList != "":
        for _, prefix := range prefixs {
            // domains Cartesian product with wordlist
            subdomain := strings.TrimSpace(prefix) + "." + item
            hosts = append(hosts, subdomain)
        }
    case iputil.IsCIDR(item):
        hosts, _ = mapcidr.IPAddresses(item)
    default:
        hosts = []string{item}
}
```

Reply:
I think it's a good idea.